package.json

```diff
@@ -51,6 +51,7 @@
   },
   "devDependencies": {
     "jasmine": "^3.4.0",
+    "nock": "13.2.9",
```

Comment:
If we're sticking with an old release, how about the most recent one, `^13.5.6`, or a comment why that release can't be used?

@na9da if you're going to release a new terriajs-server, could we get the CVEs fixed, and a Node.js 20 bump so we could use nock 14 instead?